torchvision/utils.py

```diff
@@ -208,6 +208,10 @@ def draw_bounding_boxes(
         raise ValueError("Pass individual images, not batches")
     elif image.size(0) not in {1, 3}:
         raise ValueError("Only grayscale and RGB images are supported")
+    elif (boxes[:, 0] > boxes[:, 2]).any().item() or (boxes[:, 1] > boxes[:, 3]).any().item():
```

Comment:
SGTM, this is what I would have used.

2 minor Q:

Maybe we don't need the `.item()` part?
Also, do we want this to be `>=` instead of `>` ?

Reply:
as discussed offline the most important thing is to prevent boxes in the wrong format, but degenerate boxes like `xmin == xmax` are not as important. So we're fine keeping `>`